api/modules/users/constants.py

```diff
@@ -0,0 +1,5 @@
+import uuid
+
+UNIQUE_CODE = uuid.uuid4().hex
+SUBJECT = 'Thank You for registering with us'
```

Comment:
Move this file to a new module called `email` under a file called `templates.py`

There are going to be some text changes but, you can leave them, I will update them before we merge 👍